docs/content/en/docs/setup/install-fleet-manager.md

```diff
@@ -27,7 +27,7 @@ kind load docker-image ghcr.io/kurator-dev/fleet-manager:{{< kurator-version >}}
 
 Install fleet manager into the management cluster.
 
-{{< boilerplate install-cluster-operator >}}
+{{< boilerplate install-fleet-manager >}}
 
 ## Install cluster operator from release package
```

Comment:
And same, fleet manager here

Reply:
Okay, this description also needs to be modified